draft-ietf-ice-pac.xml

```diff
@@ -116,56 +116,56 @@
           candidates are very likely to result in a valid pair.
         </t>
         <t>
-          In most ICE scenarios, the lengthy timeouts for connectivity checks,
```

Comment:
I would just say "for connectivity check transactions"

Reply:
I can do that.